tella-vault/src/main/java/com/hzontal/tella_vault/database/VaultDataSource.java

```diff
@@ -307,19 +310,21 @@ private String getFilterQuery(FilterType filter, String parentId) {
             case PHOTO:
                 return cn(D.C_MIME_TYPE) + " LIKE '" + "image/%" + "'";
             case OTHERS:
-                return cn(D.C_MIME_TYPE) + " NOT LIKE '" + "audio/%" + "'"
-                        + cn(D.C_MIME_TYPE) + " NOT LIKE '" + "video/%" + "'"
-                        + cn(D.C_MIME_TYPE) + " NOT LIKE '" + "image/%" + "'"
+                return cn(D.C_MIME_TYPE) + " NOT LIKE '" + "audio/%" + "' AND "
+                        + cn(D.C_MIME_TYPE) + " NOT LIKE '" + "video/%" + "' AND "
+                        + cn(D.C_MIME_TYPE) + " NOT LIKE '" + "image/%" + "' AND "
                         + cn(D.C_MIME_TYPE) + " NOT LIKE '" + "text/%" + "'"
                         ;
             case DOCUMENTS:
                 return cn(D.C_MIME_TYPE) + " LIKE '" + "text/%" + "' OR " + cn(D.C_MIME_TYPE) + " IN " +
                         "(" + "'application/pdf'" + ","
                         + "'application/msword'" + ","
-                        + "'application/vnd.ms-excel'" + ""
-                        + "'application/mspowerpoint'" + ""
-                        + "'audio/flac'" + ""
-                        + "'application/zip'" + ""
+                        + "'application/vnd.ms-excel'" + ","
+                        + "'application/mspowerpoint'" + ","
+                        + "'audio/flac'" + ","
+                        + "'application/zip'" + ","
+                        + "'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'" + ","
+                        + "'application/octet-stream'" + ""
```

Comment:
I think the problem here remains in the importing Function you can find it in the `MediaFileHandler` class with `importOthersUri` funtion, this should fix only the preview not the importing problem. 
What do you think @EseCoder ?

Reply:
Doesn't it import? I thought the problem was just file preview?